packages/devel/heimdal/package.mk

```diff
@@ -13,6 +13,11 @@ PKG_LONGDESC="Kerberos 5, PKIX, CMS, GSS-API, SPNEGO, NTLM, Digest-MD5 and, SASL
 PKG_TOOLCHAIN="autotools"
 PKG_BUILD_FLAGS="-parallel"
 
+pre_configure_host() {
+  # configure step misconfigures with gcc 14 unless this error is degraded to a warning
+  export CFLAGS=-Wno-error=implicit-function-declaration
```

Comment:
IMO this option has to be added to instead of overwriting the build system CFLAGS.

Reply:
How? I spent 2 hours just to figure out that heimdal code itself is fine but that actually configure script is broken. Proper fix would be to update heimdal configure scripts, but that's way out of my skills. Next best thing is to add above option to AC_LANG_PROGRAM(), which is used for checking if library exists on the system. Global CFLAGS is the only way that I found.

Reply:
`export CFLAGS+=" -Wno-error=implicit-function-declaration"`

Please test with your before failing system.

Reply:
Oh, I misunderstand what you mean. Yeah, that works too.